### PR TITLE
Feature/category

### PR DESCRIPTION
**Pull request by Clarity-89**
Update requirements, remove bower

### CODE REVIEW

File: src/conf/settings/production.py [hunk 1]
```diff
@@ -78,9 +78,6 @@
 SECURE_SSL_REDIRECT = True
 SECURE_HSTS_SECONDS = 0  # start really low
 
-SESSION_COOKIE_SECURE = True
-CSRF_COOKIE_SECURE = True
-
```
Comment (Clarity-89) [inline src/conf/settings/production.py]: Duplicate declaration.

File: composer.json [hunk 2]
```diff
@@ -5,9 +5,33 @@
   "require": {
     "tedivm/jshrink": "~1.0",
     "symfony/console": "^2.7",
-    "raven/raven": "^0.12.0"
+    "raven/raven": "^0.12.0",
+    "etki/opencart-core-installer": "^0.1.0",
```
Comment (sergei-maertens) [inline composer.json]: this is confusing me. we're ditching opencart, why is this here? :P
